src/init/AppRunner.ts

```diff
@@ -33,41 +35,65 @@ export class AppRunner {
   /**
    * Starts the server with a given config.
    * This method can be used to start the server from within another JavaScript application.
+   *
    * Keys of the `variableBindings` object should be Components.js variables.
    * E.g.: `{ 'urn:solid-server:default:variable:rootFilePath': '.data' }`.
    *
+   * `settings` corresponds to values that could be passed to the CLI.
```

Comment:
That _could_ be passed (so keys with  _default_ value), or that _are_ passed?

Reply:
Not quite sure what you mean, but passing a settings object with `{ file: './data' }` is identical as calling the server with `--file ./data`. The `could` means that the only valid keys for this object are the same keys that would be accepted when starting this config through the CLI.

Reply:
```suggestion
   * `settings` are CLI argument names and their corresponding values.
```

(So we could keep terminology consistent by talking about "[CLI] argument names", which implies that they are short names.)